packages/cli/src/commands/db.ts

```diff
@@ -68,16 +68,17 @@ const runMigrateUp = async ({silent = false} = {}) => {
   return runPrismaGeneration({silent})
 }
 
-export const runMigrate = async (name?: string) => {
+export const runMigrate = async (flags: object = {}) => {
   if (process.env.NODE_ENV === "production") {
     return runMigrateUp()
   }
+  // @ts-ignore escape:TS7053
+  const nestedFlags = Object.keys(flags).map(key => [`--${key}`, flags[key]])
+  const options = ([] as string[]).concat(...nestedFlags)
 
-  const silent = Boolean(name)
-  const args = ["migrate", "save", schemaArg, "--create-db", "--experimental"]
-  if (name) {
-    args.push("--name", name)
-  }
+  const silent = options.includes('name')
```

Comment:
change code?

`options.length > 0`

Reply:
i think there's a usecase for wanting the wizard experience but also providing custom options, I'd leave it like this